test/evergreen.yml

```diff
@@ -1688,6 +1582,93 @@ variables:
           <<: *configure_flags_with_builtins
       - func: "run workgen test"
 
+#########################################################################################
+#    Stress tests that validate disagg behavior in leader, follower, and switch modes.  #
+#########################################################################################
+
+  - &format-stress-test-disagg-leader-palm
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "format test disagg"
+        vars:
+          format_args: disagg.page_log=palm disagg.mode=leader runs.rows=100000 runs.ops=100000
+          times: 5
+
+  - &format-stress-test-disagg-leader-verify
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "format test disagg"
+        vars:
+          # FIXME-WT-15619 and FIXME-WT-15618 Remove this task when disagg verification is working reliably with deltas.
+          format_args: disagg.mode=leader ops.verify=1 disagg.leaf_page_delta=0 disagg.internal_page_delta=0
+          times: 5
+
+  - &format-stress-test-disagg-leader
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "format test disagg"
+        vars:
+          format_args: disagg.mode=leader runs.timer=5:10
+          times: 15
+
+  - &format-stress-data-validation-test-disagg-leader
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "format test disagg"
+        vars:
+          # Validate data with a non-layered table.
+          format_args: disagg.mode=leader ops.verify=1 runs.mirror=1 table1.runs.source=table table1.disagg.enabled=0 runs.timer=5:10
```

Comment:
Technically we could automate the behaviour of creating a non-disagg for when mirror is enabled. Is there a ticket tracking this work?

Reply:
Rn, we only enable mirrored tables 10% of the times by default and I wished to test the validation as much as I can. I can create a ticket for this. You'e right ideally we shouldn't need this seperate task and let things be randomly configured.

Reply:
Updated.